lib/json_web_token.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class JsonWebToken
+  class << self
+    def encode(payload, exp = 5.minutes.from_now)
+      payload[:exp] = exp.to_i
+      JWT.encode(payload, jwt_secret)
+    end
+
+    def decode(token)
+      body = JWT.decode(token, jwt_secret)[0]
+      HashWithIndifferentAccess.new body
```

Comment:
Is there a reason we need `HashWithIndifferentAccess`?

Reply:
Just allows me to use symbols to access the data after it's decoded. Is it fine to use here?